platform/stm32f0xx/platform.mk

```diff
@@ -25,7 +25,7 @@ SCRIPT_DIR := $(PLATFORM_DIR)/scripts
 CDEFINES := USE_STDPERIPH_DRIVER STM32F072 HSE_VALUE=32000000
 CFLAGS := -Wall -Wextra -Werror -g3 -Os -std=c11 -Wno-discarded-qualifiers \
 					-Wno-unused-variable -Wno-unused-parameter -Wsign-conversion -Wpointer-arith \
-					-ffunction-sections -fdata-sections -fno-builtin -flto \
+					-Wno-implicit-fallthrough -ffunction-sections -fdata-sections -fno-builtin -flto \
```

Comment:
Can you help me understand why you are are disabling compiler warnings globally?